spisea/ifmr.py

```diff
@@ -1,27 +1,677 @@
 ##########################################################
 #
 #
-#This IFMR comes from Raithel et al. 2017
+#IFMR_Raithel18 comes from Raithel et al. 2018 and has no metallicity dependence
 #https://arxiv.org/pdf/1712.00021.pdf
 #
+#IFMR_Spera15 comes from Spera et al. 2015 appendix C and includes metallicity dependence
+#https://arxiv.org/pdf/1505.05201.pdf
+#
+#Both IFMRs rely on Kalirai et al. 2007 WD IFMR on the low mass end < 9 M_sun for Raitehl18, and < 7 M_sun for Spera15
+#https://arxiv.org/pdf/0705.0977.pdf
 #
 #########################################################
 
 import numpy as np
 
-
 class IFMR(object):
+    def __init__(self):
+        pass
+
+    def get_Z(self, Fe_H):
+        """
+        This function converts metallicity given as [Fe/H] into Z values.
+        It comes from Sharma et al. 2011 "Galaxia: A Code To Generate A Synthetic Survey of The Milky Way" equation (15) 
+        https://ui.adsabs.harvard.edu/abs/2011ascl.soft01007S/abstract
+
+        """
+        return 10**(Fe_H - 1.72125)
```

Comment:
I'm not sure how important it is, but this equation assumes Z_solar = 0.019. Different evolution models have different values for Z_solar, and many recent evolution models (MIST, Ekstrom+12, Parsec) have Z_solar = 0.014 or Z_solar = 0.015. This changes the value in the exponent from -1.72125 to - 1.85387. Is this difference tolerable for the rest of the IFMR calculation?

In the evolution models, the assumed value of Z_solar is saved under <evo_model_class>.z_solar. One could imagine calling that in this function if this is important.